css/Panels.less

```diff
@@ -95,7 +95,9 @@
 		content: @moon-icon-arrowlargeleft;
 		color: @moon-spotlight-text-color;
 		-webkit-transform: translate3d(0, 0, 0);
+		transform: translate3d(0, 0, 0);
 		transition: -webkit-transform 0.2s linear, opacity 0.2s linear;
```

Comment:
This line should likely be the following:
```
-webkit-transition: -webkit-transform 0.2s linear, opacity 0.2s linear;
```
with the prefix on the value but not the rule, it would only get picked up by the rare case where the browser supports both the prefix and non prefixed versions. (This would be an existing bug, not one you introduced.)